.github/workflows/integration-test.yml

```diff
@@ -8,6 +8,7 @@ jobs:
       PORTAL_SOURCE_DIR: /home/runner/work/cbioportal/cbioportal/cbioportal
       PORTAL_COMPOSE_DIR: /home/runner/work/cbioportal/cbioportal/cbioportal-docker-compose
       PORTAL_INFO_DIR: /home/runner/work/cbioportal/cbioportal/portalInfo
+      DOCKER_IMAGE_CBIOPORTAL: cbioportal/cbioportal:demo-java-17
```

Comment:
TODO: revert this once everything is merged and cbioportal-docker-compose is updated.